lms/templates/fields/field_dropdown.underscore

```diff
@@ -4,6 +4,12 @@
     </label>
 <% } %>
 
+<% if (screenReaderTitle) { %>
+    <label class="sr" for="u-field-select-<%- id %>">
+        <%- screenReaderTitle %>
+    </label>
+<% } %>
```

Comment:
@ahsan-ul-haq @cptvitamin I am not sure about the approach here. This template is used on both the account settings page and the profile page (and in the future is going to get used in other places too). In some places the `title` is going to be shown and in other places not. If the `title` needs to be shown:
1. if we do not set `screenReaderTitle`, line 27 is going to be empty.
2. if we do set `screenReaderTitle`, we will have both the two labels above in the html.

I can think of two better approaches here:
1. We make the element for the title a non-label and have screen readers ignore it. This means the `screenReaderTitle ` will be the one read by screen readers irrespective of whether the title is being shown or now.
2. We remove the variable `screenReaderTitle` and add a variable `titleIsVisible` and then in line 2 add `<% if (!titleIsVisible) { %> class="sr" <% } %>`. This way the label on line 2 will always be the one used by screen readers but we can control its visibility with the variable `titleIsVisible`. However, this approach will only work if we expect that whenever we want to have the `title` visible, we will want screen readers to read that value as well.

I also noted this about the read-only fields: https://openedx.atlassian.net/browse/AC-100?focusedCommentId=114491&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-114491

Is it possible to have a solution which can be used in all the FieldView templates? We expect these to be used in other pages too and so it would be nice to have a single approach in all of them.

All the FieldView templates are located here: https://github.com/edx/edx-platform/tree/ahsan/AC-99-learner-profile-location-language-fields-accessibility/lms/templates/fields